.travis.yml

```diff
@@ -28,7 +28,6 @@ matrix:
           packages:
             - nginx postgresql-9.6-pgtap
       env:
-        - POD_TESTING=1
```

Comment:
The pod tests act on this value too. I don't think it can be removed.

Reply:
I'm proposing to remove the check from the two places this variable is used, as per latest commit to this PR. I think it simplifies code and it's better that all checks are consistently applied, given that these pod tests are not resource heavy.